resources/web/wwi/FloatingIde.js

```diff
@@ -0,0 +1,15 @@
+import FloatingWindow from './FloatingWindow.js';
+
+export default class FloatingIDE extends FloatingWindow {
+  constructor(parentNode, name, url) {
+    super(parentNode, name, url);
+
+    this.headerText.innerHTML = 'IDE';
```

Comment:
```suggestion
    this.headerText.innerHTML = 'Source Code Editor';
```

As we changed the tooltip from `IDE` to `Source Code Editor`, we should maybe also change the header